uwblueprint/sistering
# Implement monthly view calendar for admin scheduling UI

## Ticket link

Closes #177 



## Implementation description
* Implemented the monthly view calendar for the admin schedule UI.
* Used the FullCalendar daygrid library add-on.



## Steps to test
1. Install new dependencies and start frontend
2. Add the following snippet to `Default.tsx`, importing components/variables when necessary:
```
<Container maxW="container.xl">
    <AdminShiftCalendar events={ADMIN_SHIFT_CALENDAR_TEST_EVENTS} />
</Container>
```
3. Check that the events are displayed appropriately. Days with no events should be greyed out. "Unsaved" events have red dots and "saved" events have green dots. Should show the following:

![Screen Shot 2022-03-13 at 1 26 40 AM](https://user-images.githubusercontent.com/47638847/158048143-ef07c00d-ce6c-4569-8b00-c5311f3d15dd.png)
4. Feel free to change the test case, as long as all events of `ADMIN_SHIFT_CALENDAR_TEST_EVENTS` are in the same month.


## What should reviewers focus on?
* Clean UI, expected behaviour & styling 🖌️ 🌈 
* <ins>**_I have defeated FullCalendar's final boss._**</ins>


## Checklist
